doc/telemetry.md

```diff
@@ -6,53 +6,54 @@ The following metrics are emitted:
-| Call Counter | `datastore`, `registration_entry`, `list`   |                   | The Datastore is listing registration entries.                                        |
-| Call Counter | `datastore`, `registration_entry`, `prune`  |                   | The Datastore is pruning registration entries.                                        |
-| Call Counter | `datastore`, `registration_entry`, `update` |                   | The Datastore is updating a registration entry.                                       |
-| Call Counter | `entry`, `cache`, `reload`                  |                   | The Server is reloading its in-memory entry cache from the datastore.                 |
-| Counter      | `manager`, `jwt_key`, `activate`            |                   | The CA manager has successfully activated a JWT Key.                                  |
-| Gauge        | `manager`, `x509_ca`, `rotate`, `ttl`       | `trust_domain_id` | The CA manager is rotating the X.509 CA with a given TTL for a specific Trust Domain. |
-| Call Counter | `registration_entry`, `manager`, `prune`    |                   | The Registration manager is pruning entries.                                          |
-| Counter      | `server_ca`, `sign`, `jwt_svid`             |                   | The CA has successfully signed a JWT SVID.                                            |
-| Counter      | `server_ca`, `sign`, `x509_ca_svid`         |                   | The CA has successfully signed an X.509 CA SVID.                                      |
-| Counter      | `server_ca`, `sign`, `x509_svid`            |                   | The CA has successfully signed an X.509 SVID.                                         |
-| Call Counter | `svid`, `rotate`                            |                   | The Server's SVID is being rotated.                                                   |
-| Gauge        | `started`                                   | `version`         | The version of the Server.                                                            |
-| Gauge        | `uptime_in_ms`                              |                   | The uptime of the Server in milliseconds.                                             |
+| Type         | Keys                                           | Labels            | Description                                                                           |
+|--------------|------------------------------------------------|-------------------|---------------------------------------------------------------------------------------|
+| Call Counter | `rpc`, `<service>`, `<method>`                 |                   | Call counters over the [SPIRE Server RPCs](https://github.com/spiffe/spire-api-sdk).  |
+| Counter      | `bundle_manager`, `update`, `federated_bundle` | `trust_domain_id` | The bundle endpoint manager updated a bundle                                          |
```

Comment:
With the table re-formatting, this diff is all but worthless :) This is the line that was introduced.

Reply:
`updated a bundle` is true, but may we say that it updated `a federated bundle`?

Reply:
Good call. Fixed.